source/extended-json.rst

```diff
@@ -2,16 +2,8 @@
 Extended JSON
 =============
 
-:Spec: 587
-:Spec-ticket: SPEC-587
-:Title: Extended JSON
-:Author: Luke Lovett, David Golden
-:Spec Lead: David Golden, Jeff Yemin
-:Advisory Group: Jeff Yemin, Christian Kvalheim, Hannes Magnusson, Matt Broadstone, Jesse Davis
-:Status: Proposed
-:Type: Standards
-:Last Modified: May 21, 2021
-:Version: 2.2.0
```

Comment:
I considered keeping this as a front matter field like the unified test format's schema version, but it doesn't seem as relevant since we don't refer to this version from outside the spec (AFAIK). I made sure to add references to the original 1.0 version and bump to 2.0 in the changelog, but removed further mention of bumps to 2.x minor versions as they didn't seem relevant.